ansible_ai_connect/main/views.py

```diff
@@ -136,6 +136,7 @@ def get_context_data(self, **kwargs):
         user = self.request.user
         if user and user.is_authenticated:
             context["user_name"] = user.username
+        context["debug"] = "true" if settings.CHATBOT_DEBUG_UI == "true" else "false"
```

Comment:
This could simply be `context["debug"] = settings.CHATBOT_DEBUG_UI`.

Reply:
Thanks for your comment. This code was wrong because `settings.CHATBOT_DEBUG_UI` is boolean and cannot have "true" as a value... I have fixed it in the newest commit.